algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -119,10 +123,10 @@ && hasPotentiallyOpenRequirementToInclude(trial.recruitment()) && hasAgeGroupToI
 
     @NotNull
     @VisibleForTesting
-    static Set<String> countriesToInclude(@NotNull ClinicalTrial trial) {
+    static Set<String> filterOnRegionsToInclude(@NotNull ClinicalTrial trial, @NotNull Set<CkbRegion> regionsToInclude) {
         Set<String> countries = Sets.newHashSet();
         for (Location location : trial.locations()) {
-            if (COUNTRIES_TO_INCLUDE.contains(location.country().toLowerCase())
+            if (regionsToInclude.stream().anyMatch(region -> region.includes(location))
```

Comment:
Should you not use location.region or something?

Reply:
This is the way I've intended it. A region in our case is one country + a list of states (or provinces). A location is a specific location with a country and a state. You can check if a location is within a region with the includes method.

Regions are user defined at runtime so it doesn't really make sense to attach them to locations.